lib/about_yml/schema.json

```diff
@@ -17,48 +17,69 @@
             "description": "The type of content in the repo",
             "enum": ["app", "docs", "policy"]
         },
-        "owner_type": {
-            "type": "string",
-            "description": "Describes whether a project team, working group/guild, etc. owns the repo",
-            "enum": ["guild", "working-group", "project"]
+        "owner": {
+          "organization": {
+              "type": "string",
+              "description": "Short name for the organization that owns the project"
+          },
+          "team": {
+            "type": {
+                "type": "string",
+                "description": "Describes whether a project team, working group/guild, etc. owns the repo",
+                "enum": ["guild", "working-group", "internal-project", "client-project"]
+            },
+            "id": {
```

Comment:
@mbland The `id` is actually under `team`. The idea here is that a group could be a `working-group` but we'd also like to know which working group it is. It isn't `name` because I thought we agreed that we should be using `id` instead of `name`.